src/Control/Monad/Fork/Class.purs

```diff
@@ -1,19 +1,3 @@
-{-
```

Comment:
This seems unnecessary, since the LICENSE file is in the root of the repository, and as this has been transferred to the Contributors organization. I can add it back, however, if necessary @garyb

Reply:
I think that's fine